chandra_aca/drift.py

```diff
@@ -173,3 +174,42 @@ def get_aca_offsets(detector, chip_id, chipx, chipy, time, t_ccd):
     ddz = dz_chip - dz_pred
 
     return ddy, ddz
+
+
+def get_web_zero_offset_table():
+    """
+    Get official SOT MP zero offset aimpoint table:
+
+    :returns: zero offset aimpoint table as astropy.Table
+    """
+    ZOA = "https://icxc.harvard.edu/mp/html/aimpoint_table/zero_offset_aimpoints.txt"
```

Comment:
It saves one line of code!  :smile:, namely `global CACHE_TABLE` in your function.  Not a big deal, but the `global` statement is something I try to avoid these days (but not always..).